trino/__init__.py

```diff
@@ -10,13 +10,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from . import auth
-from . import dbapi
-from . import client
-from . import constants
-from . import exceptions
-from . import logging
-
-__all__ = ['auth', 'dbapi', 'client', 'constants', 'exceptions', 'logging']
```

Comment:
Why was this change needed?

Reply:
The revert question is why does we need this?
I remove those lines because I don't wanna add an import for every new files.

Reply:
cc: @lukasz-walkiewicz any opinions?

Reply:
Maybe, but the fact that all current modules are exposed.  
So the restriction is unnecessary.